modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/toast/commands/OpenToast.es.js

```diff
@@ -136,15 +121,16 @@ function openToast({
 			displayType={type}
 			onClick={(event) => onClick({event, onClose: onCloseFn})}
 			onClose={onCloseFn}
-			title={
-				title && (
-					<Text allowHTML={titleType === TYPES.HTML} string={title} />
-				)
-			}
 			variant={variant}
 			{...toastProps}
 		>
-			<Text allowHTML={messageType === TYPES.HTML} string={message} />
+			<div
```

Comment:
Using a `div` here since we now render the whole content, so it makes no difference what type of element we use. 

Users need to make sure that `message` is an `inline` element. If it isn't, the message will simply jump to the next line. Not the end of the world 🤷